src/bluesky/run_engine.py

```diff
@@ -1908,8 +1909,8 @@ def _close_run_trace(self, msg: Msg):
         reason = msg.kwargs.get("reason", self._reason)
         try:
             _span: Span = self._run_tracing_spans.pop()
-            _span.set_attribute("exit_status", exit_status)
-            _span.set_attribute("reason", reason)
+            _span.set_attribute("exit_status", exit_status or "None")
+            _span.set_attribute("reason", reason or "None")
```

Comment:
```suggestion
            _span.set_attribute("exit_status", exit_status if exit_status is not None else "None")
            _span.set_attribute("reason", reason if reason is not None or "None")
```

Just using "truthy" would convert `''` -> `'None'` which does not seem right to me.